### PR TITLE
B 21580 unlock gov search queues

### PR DESCRIPTION
**Pull request by KonstanceH**
## Summary
Original integration PR: https://github.com/transcom/mymove/pull/14111
KICKBACK PR: https://github.com/transcom/mymove/pull/14239

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### How to test

1. Before running make/client servers change `FEATURE_FLAG_MOVE_LOCK=true` in your envrc
2. Run `direnv allow` and then you can start up your make/client servers
4. Login as 2 different QAE, GSR, CSR, or Headquarters office users in 2 different browsers.
5. Officer 1: Search for any existing move like DRTTFR
7. Officer 1: Click record for move to see move details page
9. Officer 1: Click Milmove button or naviagte away from move(signing out, closing out the page)
11. Officer 2: Search for the same move Officer 1 opened previously
12. Officer 2: Confirm there is no lock next to the record move code

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

## Screenshots
**_Before_**
Screenshot 2024-11-04 at 11 53 14 PM

**_After_**
Screenshot 2024-11-04 at 11 43 40 PM

### CODE REVIEW

**Review by WeatherfordAaron — APPROVED** [review]
lgtm . Did not see a diff between the INT changes and this MAIN change
👽

File: pkg/handlers/ghcapi/move_test.go
Comment (taeJungCaci) [inline pkg/handlers/ghcapi/move_test.go:1]: I'm not seeing changes in https://github.com/transcom/mymove/pull/14111/files#diff-144d13c05acf08ca89cb2e5aa6e34ac1a039c43e9e5894a8c4f3ea109cee028b for this file. Could you check please?

**Review by taeJungCaci — CHANGES_REQUESTED** [review]

File: pkg/handlers/ghcapi/move_test.go
Comment (KonstanceH) [reply pkg/handlers/ghcapi/move_test.go:1]: oof yea I missed those. I'll add them in and push them up!

**Review by taeJungCaci — APPROVED** [review]
LGTM!

**Pull request merged**